src/app.js

```diff
@@ -1 +1,35 @@
-// write code here
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function moveFiles() {
+  const [srcPath, destPath] = process.argv.slice(2);
+
+  if (!srcPath || !destPath) {
+    console.error('Provide 2 correct arguments');
```

Comment:
The error message 'Provide 2 correct arguments' could be more descriptive. Consider specifying that both source and destination paths are required.